apps/web/app/api/feedback/route.ts

```diff
@@ -12,9 +13,17 @@ export async function GET(req: NextRequest) {
 
     const page = parseInt(req.nextUrl.searchParams.get("page") || "1", 10);
     const limit = parseInt(req.nextUrl.searchParams.get("limit") || "20", 10);
+    const sentiment =
```

Comment:
`function parseSentimentsQueryParam(searchParams: URLSearchParams): string[]`
+
`function transformSentimentsIntoSearchParams(sentiments: string[]): URLSearchParams`

Split and move this logic into helper functions and try use native classes here e.g. `URLSearchParams`